docs/reference/release-notes/highlights.asciidoc

```diff
@@ -13,11 +13,30 @@ For detailed information about this release, see the <<es-release-notes>> and
 
 // Use the notable-highlights tag to mark entries that 
 // should be featured in the Stack Installation and Upgrade Guide:
+
 // tag::notable-highlights[] 
-// [discrete]
-// === Heading
-//
-// Description. 
+[discrete]
+=== Better space efficiency for `keyword`, `match_only_text`, and `text` fields
```

Comment:
I think this better communicates the value (lower cost of ownership), but the current heading is fine.
```suggestion
=== Storage savings for `keyword`, `match_only_text`, and `text` fields
```